File: notify/sendToList1.py
# Documentation: https://www.twilio.com/notify/api
# Documentation: https://www.twilio.com/docs/notify/api/notifications
# ------------------------------------------------------------------------------
print("+++ Send notifications.")
import os
import json
from twilio.rest import Client
account_sid = os.environ.get("ACCOUNT_SID")
auth_token = os.environ.get("AUTH_TOKEN")
client = Client(account_sid, auth_token)
#
phoneNumber1 = os.environ.get("PHONE_NUMBER1")
#
print("+ Send notifications to: " + yourPhoneNumber)
print("+ Using notification service, SID: " + os.environ.get("NOTIFY_SERVICE_SID"))
notify_service_sid = os.environ.get("NOTIFY_SERVICE_SID")
notification = client.notify.services(notify_service_sid).notifications.create(
    body='Hello to you two.',
    to_binding=[
        json.dumps({"binding_type":"sms", "address": phoneNumber1}),
        json.dumps({"binding_type":"sms", "address": os.environ.get("PHONE_NUMBER4")})
    ]
  )
print("+ Notification SID: " + notification.sid)
print("+++ Exit.")
# ------------------------------------------------------------------------------

File: notify/sendToList1.php
<?php
require __DIR__ . '/../../twilio-php-master/Twilio/autoload.php';
use Twilio\Rest\Client;
$notifySid = getenv('MASTER_NOTIFY_SERVICE_SID');
$toPhonenumber = getenv('MY_PHONE_NUMBER');
echo "+ Send a Notify message, notification SID: " . $notifySid
        . " To: " . $toPhonenumber
        . "\xA";
$twilio = new Client(getenv('MASTER_ACCOUNT_SID'), getenv('MASTER_AUTH_TOKEN'));
$notification = $twilio->notify->v1->services($notifySid)
        ->notifications
        ->create( array(
            "body" => "hello 2.4",
            "toBinding" => array(
                "{\"binding_type\":\"sms\", \"address\":\"" . getenv('MY_PHONE_NUMBER') . "\"}"
            )
            // I couldn't get DeliveryCallbackUrl to work in this PHP program.
            // , "delivery_callbackurl" => "https://tfpecho.herokuapp.com/deliveryCallbackUrl"
        )
);
echo "+ Notification SID: " . $notification->sid . "\xA";
?>
